src/xFactor/Main.java

```diff
@@ -0,0 +1,123 @@
+package xFactor;
+
+public class Main {
+	public static void printJudges() {
+		for(int i = 0; i < Judge.getJudges().size(); i++) {
+			System.out.println(Judge.getJudges().get(i));
+		}
+	}
+	public static void main(String[] args) {
+		Judge judge1 = new Judge("Petur", "Ivanov", 36, "male", "Sofia");
+		Judge judge2 = new Judge("Elena", "Petrova", 36, "male", "Sofia");
+		Judge judge3 = new Judge("Sania", "Jekova", 42, "female", "Burgas");
+		Judge judge4 = new Judge("Milen", "Stoyanov", 50, "male", "Plovdiv");
+//		judge2.setSpecial(true);

```

Comment:
when you submit something for a review, there should not be any commented lines of code. If you really have to comment something out, you need to add a message why this is commented